config/locales/api/help/en.html

```diff
@@ -34,11 +34,24 @@
+                
+            </a>
+            <h3><span style="color: #FF3355; font-size: 90%; float: right;">NEW!</span> App for your patients</h3>
+            <p>Patients can now chart their own BPs and blood sugars with a simple app called the "BP Passport" app.</p>
+            <p>Ask your patients to go to the Android or iOS app store and install the "BP Passport" app.</p>
+            <p><a href="#" onclick="open_window('help-bp-passport-app'); return false">Learn more...</a></p>
```

Comment:
Clicking this doesn't scroll to the top of the overlay. I noticed we did the scroll magic in the progress control tab. Can we port the same magic here too?

Reply:
Yes, can you help me to do this?